lib/gem2rpm/specification.rb

```diff
@@ -13,6 +15,11 @@ def description
       Helpers.word_wrap(d, 78) + "\n"
     end
 
+    # The URL of this gem's home page
+    def homepage
+      (super.to_s.empty?) ? HOMEPAGE_EMPTY : super
```

Comment:
I am sorry, but that is definitely not something I have on my mind. This code should look like:

```ruby
BLANK_RE = /\A[[:space:]]*\z/

def homepage
  h = super
  h = nil if h && BLANK_RE =~ h
  h
end
```

The generated .spec file will then contain the URL tag only when it won't be empty.

Just FTR, this is the #blank method I referred to: http://www.rubydoc.info/gems/activesupport/String#blank%3F-instance_method